build/Tests/UnitTests/Foundation.WindowsOnly/Foundation.WindowsOnly.UnitTests.vcxproj

```diff
@@ -234,8 +234,6 @@
   </ItemGroup>
   <ItemGroup>
     <ClangCompile Include="$(StarboardBasePath)\Frameworks\Foundation\NSObject_NSCoding.mm" />
```

Comment:
VS automatically fixed this

---
In reply to: [73436698](https://github.com/Microsoft/WinObjC/pull/806#discussion_r73436698) [](ancestors = 73436698)